integration/e2e/s3_storage_client_test.go

```diff
@@ -0,0 +1,105 @@
+// +build requires_docker
+
+package e2e_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/cortexproject/cortex/integration/e2e"
+	e2edb "github.com/cortexproject/cortex/integration/e2e/db"
+	s3 "github.com/cortexproject/cortex/pkg/chunk/aws"
+	"github.com/cortexproject/cortex/pkg/util/flagext"
+)
+
+func TestS3Client(t *testing.T) {
+	networkName := "s3-test"
```

Comment:
As of today there's no need to define a dedicate network. You can just reuse the global `networkName`.